org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/AbstractNullAnnotationTest.java

```diff
@@ -81,16 +83,30 @@ protected void setUp() throws Exception {
 		setUpAnnotationLib();
 	}
 
-	/**
-	 * @deprecated indirectly uses deprecated class PackageAdmin
-	 */
 	protected void setUpAnnotationLib() throws IOException {
 		if (this.LIBS == null) {
 			String[] defaultLibs = getDefaultClassPaths();
 			int len = defaultLibs.length;
 			this.LIBS = new String[len+1];
 			System.arraycopy(defaultLibs, 0, this.LIBS, 0, len);
 			this.LIBS[len] = getAnnotationLibPath();
+			String version = this.complianceLevel >= ClassFileConstants.JDK1_8 ? "[2.0.0,3.0.0)" : "[1.1.0,2.0.0)";
```

Comment:
JDT's complianceLevel is supposed to be 1.8 or newer already so I don't think this one is  needed.

Reply:
I still see eclipse.jdt.core/org.eclipse.jdt.annotation_v1 in the repo. Can this also be removed?

Reply:
AFAIK eclipse.jdt.core/org.eclipse.jdt.annotation_v1 used to be for older Java versions but Ie3b29176203259b4c1fc947d82a2cc1e58d9e5ef by @jarthana changed it to also to Java 8. IMHO it is therefore not required anymore. @akurtakov can you check?

Reply:
I don't know which repo you speak about. I don't see it in https://download.eclipse.org/eclipse/updates/4.35-I-builds/I20241203-1800/plugins/ .

Reply:
> I don't know which repo you speak about. I don't see it in https://download.eclipse.org/eclipse/updates/4.35-I-builds/I20241203-1800/plugins/ .

https://github.com/eclipse-jdt/eclipse.jdt.core/tree/master/org.eclipse.jdt.annotation_v1

Reply:
Please open separate issue about that as it is not a prereq for this PR.

Reply:
commented out in https://github.com/eclipse-jdt/eclipse.jdt.core/blob/243787e8352163b5b12ff21d115049560a4ecfe3/pom.xml#L61 

Confusing. I personally prefer it if unused plug-ins are removed from the Git repo but of course that is something every project can decide for itself.

Reply:
> Please open separate issue about that as it is not a prereq for this PR.

Sure https://github.com/eclipse-jdt/eclipse.jdt.core/pull/3400